src/main/java/com/kobylynskyi/graphql/codegen/mapper/TypeDefinitionToDataModelMapper.java

```diff
@@ -130,6 +115,21 @@ private static ParameterDefinition merge(ParameterDefinition typeDef, ParameterD
                 .flatMap(Collection::stream)
                 .forEach(paramDef -> allParameters
                         .merge(paramDef.getName(), paramDef, TypeDefinitionToDataModelMapper::merge));
+
+
+
+        if (mappingContext.isSupportUnknownFields() ){
+            ParameterDefinition unknownFields = new ParameterDefinition();
+            unknownFields.setName(mappingContext.getUnknownFieldsPropertyName());
+            unknownFields.setOriginalName(mappingContext.getUnknownFieldsPropertyName());
+            unknownFields.setType("java.util.Map<String,Object>");
+            unknownFields.setAnnotations(Arrays.asList(
+                    JsonAnySetter.class.getName(),
+                    JsonAnyGetter.class.getName()
+            ));
```

Comment:
I see you have exact same instantiation of `ParameterDefinition` in different pleases. Please extract it.

Reply:
The logic is now externalized in `com.kobylynskyi.graphql.codegen.mapper.UnknownFieldsSupport` and used in both  `com.kobylynskyi.graphql.codegen.mapper.InputDefinitionToDataModelMapper` and  `com.kobylynskyi.graphql.codegen.mapper.TypeDefinitionToDataModelMapper`